jf_agent/jf_jira/jira_download.py

```diff
@@ -859,6 +859,26 @@ def download_customfieldoptions(jira_connection, project_ids):
     return result
 
 
+@diagnostics.capture_timing()
+@logging_helper.log_entry_exit(logger)
+def download_teams(jira_connection):
+    logger.info('downloading jira teams... [!n]')
+    server = jira_connection._options['server']
+    try:
+        teams_url = jira_connection.JIRA_BASE_URL.format(
+            server=server,
+            rest_path='teams-api',
+            rest_api_version='1.0',
+            path='team'
+        )
+        teams = jira_connection._get_json('team', base=teams_url)
```

Comment:
Nit: Use `retry_for_429s` here to handle rate-limiting

Reply:
`teams = retry_for_429s(jira_connection._get_json, 'team', base=teams_url)`

Reply:
https://github.com/Jellyfish-AI/jf_agent/pull/284/commits/471e515142bd0dd50c10ff5c682474e3b259436b